scipy/sparse/csgraph/_laplacian.py

```diff
@@ -69,6 +70,20 @@ def laplacian(
         sparse matrices unless the sparsity pattern is symmetric or
         `form` is 'function' or 'lo'.
         Default: False, for backward compatibility.
+    variant : 'repelling', 'opposing' or 'unsigned'
+        Specifies the method used to compute vertex degrees.
+        All variants are the same in the case of non-negative edge weights.
+        Detailed discussions on the reasoning behind each type of variant
+        can be found in [2].:
+
+        * 'repelling' is standard degree calculation using edge weights.
+        * 'opposing' computes the degree using absolute values of edge weights.
+          This prevents cancellation of positive and negative edge weights in
+          signed graphs, as described in [3].
+        * 'unsigned' discards the sign of edge weights.
+          This removes all the information contained in the sign.
```

Comment:
This description talks about calculation of degree values. But not the sign used in the definition of L itself. I think this will be confusing for someone coming to this function who has never heard of or cares about signed graphs. 

Do I understand correctly that in addition to the treatment of signs when computing degree, that some of these variants compute `L=D-A` and others compute `L=D+A`?  And maybe it would help if this description started with some indication that this keyword handles variants for signed graphs. That will help folks who are using standard edge weights figure out that they don't need to worry about this keyword arg.